index.html

```diff
@@ -0,0 +1,68 @@
+<!--HTML page created for learning purposes-->
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <title id="title">Geena's practice HTML file</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        
+        <div class="banner note">
+          <h1>Using paragraphs</h1>
+          <p>Just testing a div p </p>
+        </div>
+        <p>
+            This page is for practicing on using the following:
+            <ol>
+                <li class="para">Paragraphs</li>
+                <li class="links">Links</li>
+                <li class="ima">Images</li>
+                <li>Spans</li>
+                <li>Tables</li>
+                <li>Lists</li>
+            </ol>
+        </p>
+        <h2 class="links">Using links</h2>
+        <a href="https://www.google.com" target="_top"> This is an example link to Google. 
+          <br>
+        This is an example link to <a href="https://www.yahoo.com" target="_blank"> Yahoo. </a>
```

Comment:
it's a bit weird to have a break tag (`<br>`) in the middle of an anchor (`<a>`) text
it might be preference, but I'd prefer this:
```
<a ...>
   ...
<a/>